src/agent.py

```diff
@@ -50,7 +50,7 @@ def __init__(
         self._col = col
         self._body_radius = body_radius
         self._sense_radius = sense_radius
-        self._body_temp = body_temp
+        self._body_temp = np.full(shape=(2*body_radius-1,2*body_radius-1), fill_value=body_temp, dtype=float)
```

Comment:
Looks like this produces a square of heat instead of a diamond shape. Is that okay?